tools/next-version

```diff
@@ -21,7 +21,7 @@ const calculateNewVersion = () => {
   const versionNameCounter = year === versionNameParts[0] && month === versionNameParts[1] ? versionNameParts[2] + 1 : 0
   const newVersionName = `${year}.${month}.${versionNameCounter}`
 
-  if (typeof versionCode !== 'number') {
+  if (versionCode && typeof versionCode !== 'number') {
```

Comment:
actually this change is unneccessary, but if you still feel that this is more consitent you can leafe it as it is

Reply:
The aim is to have exactly the same script for native and webapp, so I would prefer to leave it as it is. When we switch to a mono repo, we can use the same script.